public/scripts/worktrollyUI.js

```diff
@@ -136,4 +137,21 @@ $("#submitNewSprint").click(function() {
         newPage = "dashboard";
         uiLoader();
     });
+});
+
+$("#logWorkSubmit").click(function() {
+    var sprintNumber = $("#sprintNumber").val();
+    var logTaskId = $("#logTaskId").val();
+    var logWorkDone = $("#logWorkDone").val();
+    var logWorkComment = $("#logWorkComment").val();
+    var logWorkStatus = $("#logWorkStatus").val();
+    logWorkDone = parseInt(logWorkDone) + parseInt(logWorkRT);
```

Comment:
This is not needed, logWorkDone is going to contain the percentage of work completed, which will be always overwritten in db.